instance/tests/test_person.py

```diff
@@ -0,0 +1,28 @@
+from global_test_case import GlobalTestCase as TestCase
+from instance.models import PopoloPerson, WriteItInstance
+from contactos.models import ContactType, Contact
+
+
+class TestPopoloPerson(TestCase):
+    def setUp(self):
+        super(TestPopoloPerson, self).setUp()
+        self.person = PopoloPerson.objects.get(id=1)
+        self.contact_type, created = ContactType.objects.get_or_create(
+            name="e-mail", defaults={"label_name": "Electronic Mail"}
+        )
+        writeitinstance = WriteItInstance.objects.get(id=1)
+        self.contact = Contact.objects.create(
+            contact_type=self.contact_type,
+            value="test@email.com",
+            person=self.person,
+            writeitinstance=writeitinstance,
+        )
```

Comment:
I think we should prefer to use json fixtures over setup code like this because I think it's easier to read - what do you think?

I think one reason I think it's easier to read is that it's then pretty easy to have different fixtures for tests that need different data - i.e. each of these cases would have their own fixture.

Not ready to make a decision on this right now, just wanna hear your thoughts.